app/apollo-provider.tsx

```diff
@@ -0,0 +1,47 @@
+'use client'
```

Comment:
Maybe now we should make provider directory? Because we have two provider file. `Providers.tsx` and `apollo-provider.tsx`. I think we should change the file name of Provider.tsx if you make this new provider file 👀

Reply:
Good idea about the `provider` directory! But `Provider.tsx` is just a renamed `chakra-provider.tsx`.
Maybe we can just rename it for clarity instead? What do you think? 👀

Reply:
Yeah, that's good! You can rename the providers to chakra provider and make provider directory!